src/averageAges.js

```diff
@@ -14,12 +14,22 @@
  *
  * @return {number}
  */
+const MALE_SEX = 'm';
+const FEMALE_SEX = 'f';
+const YEARS_IN_CENTURY = 100;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menList = people.filter((person) => {
+    return century
+      ? person.sex === MALE_SEX
+        && Math.ceil(person.died / YEARS_IN_CENTURY) === century
+      : person.sex === MALE_SEX;
+  });
+  const menAvgAge = menList.reduce((prev, person) => (
+    prev + (person.died - person.born)
```

Comment:
It would be great to create a function-helper for calculating an average age as this logic is the same for men and women